### ISSUE TITLE
Win key symbol not displayed in Remontoire

### ISSUE THREAD

**Issue by jsy1001** [regolith-linux/regolith-desktop#488]
- [X] I've [run a search on the existing bug reports](https://help.github.com/en/github/searching-for-information-on-github/searching-issues-and-pull-requests) to make sure I'm not filing a duplicate
- [X] I've read [the Regolith documentation](https://regolith-linux.org/docs/) and didn't find any relevant topic
- [ ] I tried [asking on Slack](https://regolith-linux.org/interact/) (especially for configuration related issues)

**Describe the bug**
In Remontoire, the symbol for the win key does not display properly. I see a narrow empty box rather than the correct symbol shown in the screenshots on regolith-linux.org.

**To Reproduce**
- Install Regolith using the PPA
- Login to a Regolith session
- Toggle Remontoire using $mod+?

**Expected behavior**
I expected Remontoire to look like the screenshots on regolith-linux.org, with key bindings shown using a win key symbol.

**Screenshots**
![Screenshot from 2020-10-08 21-02-54](https://user-images.githubusercontent.com/4949517/95508076-58923800-09aa-11eb-82cb-979a78b14091.png)

**Configuration file(s)**
I can reproduce the problem with the out-of-the-box configuration.

**Installation Details**
 - Regolith Install Type: PPA
 - Regolith Version: 1.4.1
 - PPA url: ppa:regolith-linux/release
 - Host OS (for PPA): Ubuntu 18.04

**Additional context**
In case its relevant - in gnome-tweak, my fonts are configured as follows:

- Window Title: Ubuntu 11
- Interface: Ubuntu Regular 11
- Document: Sans Regular 11
- Monospace: Monospace Regular 12

I don't know how to check which font is actually being used to display the symbol in question.

**Comment by kgilmer** [comment]
Hi @jsy1001 , the font with the windows icon comes from `FontAwesome`.  You should have that installed if you installed Regolith, but perhaps it's good to check.  It may also be that you have some other font that is overriding fontawesome when Remontoire renders the window.  There is an [open bug](https://github.com/regolith-linux/remontoire/issues/6) to explicitly define the font used, that I have yet to fix.  I would expect that the next version of Remontoire will resolve this issue.

**Comment by jsy1001** [comment]
Hi @kgilmer , I do indeed have `FontAwesome` installed.

I found the relevant character in the comments in the i3 config. This character is U+F17A, and the corresponding glyph in the private use area of `FontAwesome` is the correct windows icon.

Your suggestion that some other font is overriding fontawesome seems plausible. Can you think of a way of identifying which one, other than trial and error?

**Comment by kgilmer** [comment]
In the open bug I listed above there is a list of the fonts I have installed.  You may diff your fonts against that and see if there is a clue to the offending font there.  FWIW I'm looking at fixing this correctly by referencing the font explicitly in the app.

**Comment by kgilmer** [comment]
I believe I may have a fix for this.  Can you run remontoire with the attached stylesheet, like:
```
./src/remontoire -t <path to style.css> -s $(printenv I3SOCK)
```

The stylesheet is available here: https://github.com/regolith-linux/remontoire/blob/master/data/style.css

**Comment by jsy1001** [comment]
I've tried running remontoire with the stylesheet as you suggest, and this does indeed fix the problem. Thank you.

**Comment by cdelnooz** [comment]
Just found this issue through search, I encountered the same problem on Ubuntu 20.04  but only after switching the looks to regolith-look-nord, out of the box, the glyphs were correct. Also for me, running with the style sheet solves the problem.

**Comment by moritzheiber** [comment]
This has been shipped with the latest release in 1.5.1
